pokemon-checker/src/components/PokemonDisplay/DisplayFunctionalComponents/AbilityDisplay.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import { AbilityDTO } from "../../../DataTransferObjects/AbilityDTO";
+import { ListItem, Popover } from "@mui/material";
+
+type AbilityDisplayProps = {
+  abilities: AbilityDTO[];
+}
+
+//TODO: (Geoff) create custom styling for name
+export const AbilityDisplay = ( { abilities }: AbilityDisplayProps) => {
+  const [anchorEl, setAnchorEl] 
+    = React.useState<HTMLElement | null>(null);
+  const [hoveredText, setHoveredText] = React.useState<string | null>(null);
+
+  const handlePopoverOpen 
+    = (event: React.MouseEvent<HTMLElement>, hovered: string) => {
+    setHoveredText(hovered);
+    setAnchorEl(event.currentTarget);
+  }
+
+  const handlePopoverClose = () => {
+    setAnchorEl(null);
+  };
+
+  const abilityHovered = Boolean(anchorEl);
+  return (
+    <div className="ability-list">
```

Comment:
Nitpick, but it might be a good idea to use `<ul>` or `<ol>` instead of `<div>` for semantics.

Reply:
right, I'll take a look at MUI to see some listing styles, one of the tasks I want to do is CSS work for the whole look, maybe add some appropriate borders, global colors, themes etc, if we do another meeting it should be a big topic to try and hammer out a style with colors, fonts etc

Reply:
![image](https://user-images.githubusercontent.com/3722350/160301666-b9782ca8-567b-4545-8c2c-c2e0d764fd57.png)

right now the div is in place of a container, i'll look for more appropriate containers for abilities, as well as styling for the popup (right now its on hover, but I want to change it to an onclick)